### PR TITLE
User crud operation

### PR DESCRIPTION
**Pull request by mukeshsin**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+//Token secret key
```
Comment (meanstack9716) [inline .env:1]: do not push .env to git @mukeshsin

File: controllers/role.controller.js [hunk 2]
```diff
@@ -1,78 +1,71 @@
...
-
-
 export const roleList = async (req, res) => {
-    try {
-      const roles = await Role.findAll();
-      console.log(roles)
-      res.status(200).send(roles);
+  try {
+    const roles = await Role.findAll();
+    console.log(roles);
```
Comment (meanstack9716) [inline controllers/role.controller.js:15]: remove console @mukeshsin

File: controllers/user.controller.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  try {
+    const { emailId, password } = req.body;
+    if (!emailId || !password) {
+      res.status(401).send({ message: "please fill the information" });
+    }
+    const salt = await bcrypt.genSalt();
+    const hashedPassword = await bcrypt.compare(req.body.password, salt);
+    const user = await User.findOne({
+      where: {
+        emailId: emailId,
```
Comment (meanstack9716) [inline controllers/user.controller.js:42]: change emailId to email @mukeshsin

File: controllers/user.controller.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  } catch (error) {
+    console.log(error);
+    res.status(500).send({ message: "500 error to the user" });
+  }
+};
+
+//create User
+export const createUser = async (req, res) => {
+  try {
+    console.log("createUser");
```
Comment (meanstack9716) [inline controllers/user.controller.js:63]: remove console @mukeshsin

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

File: controllers/user.controller.js (hunk 4 above)
Comment (mukeshsin) [reply controllers/user.controller.js:63]: ok sir

File: controllers/user.controller.js (hunk 3 above)
Comment (mukeshsin) [reply controllers/user.controller.js:42]: ok sir i  do rename

File: .env (hunk 1 above)
Comment (mukeshsin) [reply .env:1]: you mean to say sir i should use gitignore in this type of case??
@meanstack9716

**Pull request merged**